### DISCUSSION TITLE
Pwnagotchi 2.8.6 Web UI or BT Connection not working

### DISCUSSION THREAD

**Discussion by Eception** [jayofelony/pwnagotchi#85] [Q&A]
Hello,

i cant access Web.UI . Says error name not resolved but dns in pwnagotchi is 8.8.8.8. Connection via usb0 and i can connect via SSH.
Plugin is enabled.

Bluetooth does not work either via config.toml file with the correct code or via bluetoothctl directly via Pwnagotchi.
Connection is there for like 1s and then error br-connection-profile-unavailable

Any tips?

**Comment by jayofelony** [comment]
http://10.0.0.2:8080 is webui address

Bluetooth is rather finicky to set up

**Comment by wpa-2** [comment]
> Hello,
> 
> i cant access Web.UI . Says error name not resolved but dns in pwnagotchi is 8.8.8.8. Connection via usb0 and i can connect via SSH. Plugin is enabled.
> 
> Bluetooth does not work either via config.toml file with the correct code or via bluetoothctl directly via Pwnagotchi. Connection is there for like 1s and then error br-connection-profile-unavailable
> 
> Any tips?

https://pwnagotchi.org/common-issues/index.html#instructions

Bluetooth does work as I set it up yesterday.
